common/models/admin-message.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "AdminMessage",
+    "base": "PersistedModel",
+    "idInjection": true,
+    "options": {
+      "validateUpsert": true
+    },
+    "mixins": {
+      "LogOnDelete": true
+    },
+    "properties": {
+      "message": {
+        "type": "string"
+      },
+      "timeStamp": {
+        "type": "date",
+        "required": true
```

Comment:
Tilføj gerne en default til timestamp

     ``` "default": "$now"````
Og overvej at kald den timeCreated i stedet for, da dette er brugt flere andre steder.

Reply:
Det har jeg fixet i det seneste commit:)